app/celer-sim/RunnerInput.hh

```diff
@@ -52,6 +52,11 @@ struct RunnerInput
     std::string physics_file;  //!< Path to ROOT exported Geant4 data
     std::string event_file;  //!< Path to input event data
 
+    // Optional setup options when event_file is a ROOT file that should be
+    // used for sampling combinations of events as opposed to just reading them
+    size_type num_sampled_events{};  //!< Events to be sampled from ROOT file
+    size_type num_clumped_events{};  //!< Events to be joined from ROOT file
```

Comment:
Since these options are specific to the ROOT "sampling" input, perhaps it would be better to have a separate struct for managing them?

Reply:
I think it would improve clarity to have
```c++
struct EventFileSampling {
size_type num_events{}; //!< Total number of events to sample
size_type num_merged{}; //!< ROOT file events per sampled event
explicit operator bool() const { return num_events > 0; }
};
```
and
```c++
EventFileSampling event_file_sampling;
```
so you can write `if (input. event_file_sampling)` and then dig into the values.